src/main/resources/liquibase/crop_changelog/22_3_0.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="cuenyad" id="v22.3.0-1">
+		<preConditions onFail="MARK_RAN">
+				<sqlCheck expectedResult="1">
+					SELECT count(1) > 0 FROM CVTERMPROP WHERE CVTERM_ID = 8201 AND value = 'Germplasm Descriptor';
+				</sqlCheck>
+		</preConditions>
+		<comment>Recategorize OBS_UNIT_ID as Experimental Design variable</comment>
+		<sql dbms="mysql" splitStatements="true">
+			set @OBS_UNIT_ID = (select cvterm_id from cvterm where name ='OBS_UNIT_ID' and cv_id = 1040 and is_obsolete = 0);
```

Comment:
@cuenyad we dont need to check by is_obsolete now as it was removed from the unique key

Reply:
done!.